tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/utils/ProjectHasNature.java

```diff
@@ -45,6 +46,13 @@ public boolean test() {
 		pd.open();
 		new WaitUntil(new ShellIsAvailable("Properties for " + projectName), TimePeriod.DEFAULT);
 		new DefaultTreeItem("Project Facets").select();
+		if (!getJavaVersion().equals("1.8")) {
+			try {
+				new WaitUntil(new ShellIsAvailable("Setting Java Build Path"), TimePeriod.DEFAULT);
+				new PushButton("Apply").click();
+			} catch (Exception e) {
```

Comment:
You are using general exception, you should be using as more specific exception as possible, in this case it could be CoreLayerException.

Reply:
Fixed.